doc/life_of_a_language_feature.md

```diff
@@ -46,22 +52,50 @@ significant changes.
 ## Acceptance and implementation
 
 If consensus is reached on a specific proposal and we decide to accept it, a
-member of the language team will be chosen to shepherd the implementation.  An
-implementation plan document will be created in a named sub-directory of the
-`accepted` folder with details about the implementation plan and **a link to the
-single canonical writeup of the language feature** which is to serve as the
-implementation reference.  The implementation plan should generally include at
-least:
+member of the language team will be chosen to shepherd the implementation.
+The implementation will be tracked via three artifacts:
+
+  - An implementation plan document
+
+  - A 'language-implementation' issue.
+
+  - A feature specification document
+
+The implementation plan must be located in a sub-directory with the name of the
+feature located inside the `accepted/future-releases/` folder. The filename should be
+`[feature-name]-implementation-plan.md`. The implementation plan should
```

Comment:
I'd prefer not to name mangle *and* use a sub-directory.  That is, accepted/future-releases/tuples/tuples-implementation-plan.md is redundant: I'd rather drop either the sub-directory or the name-mangling, and I'd rather drop the name-mangling because I want to make it easy to have supporting documents and artifacts.